更新完之后，提示“当前QQ版本过低，请升级至最新版本后再登录。”

### 问题描述

更新完毕之后发生。尝试过删除bot文件夹的cache目录，无用。

### 复现

更新完毕之后发生。

### mirai-core 版本

2.13.4

### bot-protocol

ANDROID_PHONE

### 其他组件版本

Running MiraiConsole v2.13.4, built on 2023-01-21 21:19:40.
Frontend Terminal: version 2.13.4, provided by Mamoe Technologies

Permission Service: Built In Permission Service

Plugins: MCL Addon v2.1.1

Object Pending Finalization Count: 0
                 committed |  init   |  used   |  max  
    Heap Memory:  100.0MB  | 186.0MB | 46.88MB | 2.89GB
Non-Heap Memory:  78.06MB  | 7.31MB  | 76.34MB |   -1  


### 系统日志

```text
kris@KrisTHL181:~/Mirai/MCL$ ~/java/jdk-17.0.5+8/bin/java -jar mcl.jar -u
  18:27:51 [INFO] iTX Technologies Mirai Console Loader version 2.1.2-61c8bd8
  18:27:51 [INFO] Runtime: OpenJDK 64-Bit Server VM 17.0.5 (arch: 64)
  18:27:51 [INFO] https://github.com/iTXTech/mirai-console-loader
  18:27:51 [INFO] This program is licensed under GNU AGPL v3
  18:27:52 [INFO] Mirai Console Loader Announcement:
Mirai Console Loader 公告栏

如果在图片上传的时候遇到问题请与我们联系 (需要提供图片文件源本)
`- 如 Unsupported image type for ExternalResource *
`  considering use gif/png/bmp/jpg format.
`- Tracker: https://github.com/mamoe/mirai/issues/new/choose

常用资源整合
`- https://mirai.mamoe.net/topic/653
 
MCL 已推出 2.1.0，更好的支持 Mirai 2.11 的插件系统，建议更新。

  18:27:52 [INFO] Verifying "net.mamoe:mirai-console" v2.13.4
  18:27:53 [INFO] Verifying "net.mamoe:mirai-console-terminal" v2.13.4
  18:27:54 [INFO] Verifying "net.mamoe:mirai-core-all" v2.13.4
  18:27:54 [INFO] Verifying "org.itxtech:mcl-addon" v2.1.1
2023-01-23 18:27:56 I/main: Starting mirai-console...
2023-01-23 18:27:56 I/main: 

==================================[ Mirai consosle 2.13.4 ]===================================
 __       __ __                   __  ______                                      __
|  \     /  \  \                 |  \/      \                                    |  \
| ▓▓\   /  ▓▓\▓▓ ______   ______  \▓▓  ▓▓▓▓▓▓\ ______  _______   _______  ______ | ▓▓ ______
| ▓▓▓\ /  ▓▓▓  \/      \ |      \|  \ ▓▓   \▓▓/      \|       \ /       \/      \| ▓▓/      \
| ▓▓▓▓\  ▓▓▓▓ ▓▓  ▓▓▓▓▓▓\ \▓▓▓▓▓▓\ ▓▓ ▓▓     |  ▓▓▓▓▓▓\ ▓▓▓▓▓▓▓\  ▓▓▓▓▓▓▓  ▓▓▓▓▓▓\ ▓▓  ▓▓▓▓▓▓\
| ▓▓\▓▓ ▓▓ ▓▓ ▓▓ ▓▓   \▓▓/      ▓▓ ▓▓ ▓▓   __| ▓▓  | ▓▓ ▓▓  | ▓▓\▓▓    \| ▓▓  | ▓▓ ▓▓ ▓▓    ▓▓
| ▓▓ \▓▓▓| ▓▓ ▓▓ ▓▓     |  ▓▓▓▓▓▓▓ ▓▓ ▓▓__/  \ ▓▓__/ ▓▓ ▓▓  | ▓▓_\▓▓▓▓▓▓\ ▓▓__/ ▓▓ ▓▓ ▓▓▓▓▓▓▓▓
| ▓▓  \▓ | ▓▓ ▓▓ ▓▓      \▓▓    ▓▓ ▓▓\▓▓    ▓▓\▓▓    ▓▓ ▓▓  | ▓▓       ▓▓\▓▓    ▓▓ ▓▓\▓▓     \
 \▓▓      \▓▓\▓▓\▓▓       \▓▓▓▓▓▓▓\▓▓ \▓▓▓▓▓▓  \▓▓▓▓▓▓ \▓▓   \▓▓\▓▓▓▓▓▓▓  \▓▓▓▓▓▓ \▓▓ \▓▓▓▓▓▓▓


2023-01-23 18:27:56 I/main: Backend: version 2.13.4, built on 2023-01-21 21:19:40.
2023-01-23 18:27:56 I/main: Frontend Terminal: version 2.13.4, provided by Mamoe Technologies
2023-01-23 18:27:56 I/main: Welcome to visit https://mirai.mamoe.net/
2023-01-23 18:27:56 W/stderr: SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
2023-01-23 18:27:56 W/stderr: SLF4J: Defaulting to no-operation (NOP) logger implementation
2023-01-23 18:27:56 W/stderr: SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
2023-01-23 18:27:57 I/plugin: Successfully loaded plugin MCL Addon v2.1.1
2023-01-23 18:27:57 I/main: Prepared built-in commands: autoLogin, help, login, logout, permission, status, stop
2023-01-23 18:27:57 I/MCL Addon: iTXTech MCL Version: 2.1.2-61c8bd8
2023-01-23 18:27:57 W/MCL Addon: iTXTech Soyuz 未安装，Soyuz MCL Handler 特性已禁用
2023-01-23 18:27:57 I/main: 1 plugin(s) enabled.
2023-01-23 18:27:58 I/WindowHelperJvm: Mirai 正在使用桌面环境. 如遇到验证码将会弹出对话框. 可添加 JVM 属性 `mirai.no-desktop` 以关闭.
2023-01-23 18:27:58 I/WindowHelperJvm: Mirai is using desktop. Captcha will be thrown by window popup. You can add `mirai.no-desktop` to JVM properties (-Dmirai.no-desktop) to disable it.
2023-01-23 18:27:58 I/main: Auto-login 3581338560, protocol: ANDROID_PHONE, heartbeatStrategy: STAT_HB
2023-01-23 18:27:58 W/stderr: ERROR StatusLogger Log4j2 could not find a logging implementation. Please add log4j-core to the classpath. Using SimpleLogger to log to the console...
2023-01-23 18:27:59 I/Bot.3581338560: Calculating type 2 PoW, it can take some time....
2023-01-23 18:27:59 I/Bot.3581338560: Got PoW result, cost: 151 ms
2023-01-23 18:28:11 E/main: net.mamoe.mirai.network.WrongPasswordException: Error(bot=Bot(3581338560), code=235, title=温馨提示, message=当前QQ版本过低，请升级至最新版本后再登录。点击进入下载页面, errorInfo=)
net.mamoe.mirai.network.WrongPasswordException: Error(bot=Bot(3581338560), code=235, title=温馨提示, message=当前QQ版本过低，请升级至最新版本后再登录。点击进入下载页面, errorInfo=)
	at net.mamoe.mirai.internal.network.components.SsoProcessorImpl$SlowLoginImpl.doLogin(SsoProcessor.kt:311)
	at net.mamoe.mirai.internal.network.components.SsoProcessorImpl$SlowLoginImpl$doLogin$1.invokeSuspend(SsoProcessor.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.internal.ScopeCoroutine.afterResume(Scopes.kt:33)
	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)

2023-01-23 18:28:11 I/Bot.3581338560: Bot cancelled: Bot closed
2023-01-23 18:28:11 I/main: mirai-console started successfully.
```


### 网络日志

```text
2023-01-23 18:27:58 V/Net 3581338560: No server list cached.
2023-01-23 18:27:58 I/Net 3581338560: ECDH key is invalid, start to fetch ecdh public key from server.
2023-01-23 18:27:59 I/Net 3581338560: Successfully fetched ecdh public key from server.
2023-01-23 18:27:59 V/Net 3581338560: Send: wtlogin.login(9:password-login)
2023-01-23 18:28:00 V/Net 3581338560: Recv: wtlogin.login
2023-01-23 18:28:11 V/Net 3581338560: Send: wtlogin.login(2:submit-slider)
2023-01-23 18:28:11 V/Net 3581338560: Recv: wtlogin.login
2023-01-23 18:28:11 W/Net 3581338560: Exception in resumeConnection.
net.mamoe.mirai.internal.network.components.LoginFailedExceptionAsNetworkException: Error(bot=Bot(3581338560), code=235, title=温馨提示, message=当前QQ版本过低，请升级至最新版本后再登录。点击进入下载页面, errorInfo=)
	at net.mamoe.mirai.internal.network.handler.CommonNetworkHandler$StateConnecting$startState$2.invokeSuspend(CommonNetworkHandler.kt:248)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.internal.ScopeCoroutine.afterResume(Scopes.kt:33)
	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
Caused by: net.mamoe.mirai.network.WrongPasswordException: Error(bot=Bot(3581338560), code=235, title=温馨提示, message=当前QQ版本过低，请升级至最新版本后再登录。点击进入下载页面, errorInfo=)
	at net.mamoe.mirai.internal.network.components.SsoProcessorImpl$SlowLoginImpl.doLogin(SsoProcessor.kt:311)
	at net.mamoe.mirai.internal.network.components.SsoProcessorImpl$SlowLoginImpl$doLogin$1.invokeSuspend(SsoProcessor.kt)
	... 9 more

2023-01-23 18:28:11 W/Net 3581338560: Network selector received exception, closing bot. (net.mamoe.mirai.internal.network.components.LoginFailedExceptionAsNetworkException: Error(bot=Bot(3581338560), code=235, title=温馨提示, message=当前QQ版本过低，请升级至最新版本后再登录。点击进入下载页面, errorInfo=))
```


### 补充信息

尝试过使用其他ISSUE的解决方法，皆无用。

--- a/mirai-core/src/commonMain/kotlin/utils/MiraiProtocolInternal.kt
+++ b/mirai-core/src/commonMain/kotlin/utils/MiraiProtocolInternal.kt
@@ -33,30 +33,30 @@ internal class MiraiProtocolInternal(
             protocols[protocol] ?: error("Internal Error: Missing protocol $protocol")
 
         init {
-            //Updated from MiraiGo (2023/1/12)
+            //Updated from MiraiGo (2023/3/7)
             protocols[MiraiProtocol.ANDROID_PHONE] = MiraiProtocolInternal(
                 "com.tencent.mobileqq",
-                537143097,
-                "8.9.23.9425",
-                "6.0.0.2530",
+                537151682,
+                "8.9.33.10335",
+                "6.0.0.2534",
                 150470524,
                 0x10400,
                 16724722,
                 "A6 B7 45 BF 24 A2 C2 77 52 77 16 F6 F3 6E B6 8D",
-                1640921786L,
+                1673599898L,
                 19,
             )
-            //Updated from MiraiGo (2023/1/12)
+            //Updated from MiraiGo (2023/3/7)
             protocols[MiraiProtocol.ANDROID_PAD] = MiraiProtocolInternal(
                 "com.tencent.mobileqq",
-                537142586,
-                "8.9.23.9425",
-                "6.0.0.2530",
+                537151218,
+                "8.9.33.10335",
+                "6.0.0.2534",
                 150470524,
                 0x10400,
                 16724722,
                 "A6 B7 45 BF 24 A2 C2 77 52 77 16 F6 F3 6E B6 8D",
-                1640921786L,
+                1673599898L,
                 19,
             )
             protocols[MiraiProtocol.ANDROID_WATCH] = MiraiProtocolInternal(

